src/Goerli-DssSpell.t.sol

```diff
@@ -279,7 +279,7 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
```

Comment:
housekeeping: we can remove L179/180 as specific of a previous spell and we can leave a comment for future need (`// Insert spell-only addresses here`).